nxdrive/data/i18n/i18n.json

```
@@ -31,7 +31,7 @@
     "CONFLICT": "Conflict",
     "CONFLICTS_AND_ERRORS": "Conflicts and errors",
     "CONFLICTS_SYSTRAY": "%1 conflict(s)",
-    "CONFLICT_ON_FILE": "The file „%1“ has a conflict",
+    "CONFLICT_ON_FILE": "The file "%1" has a conflict",
```

All double quotes must be escaped ;)
```suggestion
    "CONFLICT_ON_FILE": "The file \"%1\" has a conflict",
```

You can see that the "translations" check is red, if you need to verifiy al is good.